internal/runtime/handler.go

```diff
@@ -506,6 +519,25 @@ func (h *Handler) getFilters(req *http.Request) dbmodel.InstanceFilter {
 	return filter
 }
 
+func (h *Handler) applyBindings(p *pkg.RuntimeDTO) error {
```

Comment:
So we read all bindings for given instance and add it to RuntimeDTO.
Could we consider other verb than `apply`?